packages/components/src/components/Paragraph/index.native.tsx

```diff
@@ -29,20 +31,16 @@ interface Props {
     children: React.ReactNode;
     className?: string;
     size?: string;
+    fontSize: number;
```

Comment:
probably should be defined only for styled component (`const Paragraph = styled.View`) otherwise when the props `fontSize` is passed to `P` component it will override `fontSize={P_SIZES[size]}` basically allowing to set any size outside of defined `P_SIZES` which is not necessary bad, just not consistent with web counterpart.

Reply:
Not true. You need to set the font size on the Text component, setting it on the View component does not affect the font size in Text component.
The `fontSize` prop accepts only ['small', 'medium', 'large', 'xlarge'] values.
`P` is a styled.View and inside it is styled.Text which takes the fontSize from the P_SIZES array.